src/crm/dynamicsWebApi.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios-observable';
+import { map } from 'rxjs/operators';
+import { lastValueFrom } from 'rxjs';
+import { AxiosError } from 'axios';
+import { getToken } from './getToken';
+import { DynamicsTestStation } from '../Interfaces/DynamicsTestStation';
+
+import { DynamoTestStation, createDynamoTestStation } from '../Interfaces/DynamoTestStation';
+
+const onRejected = (error: AxiosError) => {
+  // Removes bearer token from being logged in the error
+  delete error.config.headers;
+  return Promise.reject(error);
+};
+
+const getTestStationEntities = async (requestUrl: string): Promise<DynamoTestStation[]> => {
+  const accessToken = await getToken();
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  axios.defaults.headers.common.Authorization = `Bearer ${accessToken}`;
+
+  axios.interceptors.response.use((response) => response, onRejected);
+
+  interface ApiFormat {
+    value: DynamicsTestStation[];
+  }
+
+  return lastValueFrom(
+    axios.get<ApiFormat>(requestUrl).pipe(
+      map((data) => data.data),
+      map((data: ApiFormat) => data.value.map((obj) => createDynamoTestStation(obj))),
```

Comment:
Can this be split out into two lines to follow the pattern of the preceding line?

Reply:
as in move everything after data.value.map( to a new line?

Reply:
`
map((data: ApiFormat) => data.value),
map((data) => createDynamoTestStation(data)),
`

Reply:
That should be two lines but github didn't make it so 😆

Reply:
I don't believe this will be possible with the two types of map used within this pipe method. Using .map applies the createDynamoTestStation to each index of data.value whereas using map like you suggested would keep data as an array which createDynamoTestStation doesn't expect. 
![image](https://user-images.githubusercontent.com/92087051/151159866-99d08f6e-0d0b-4ff8-891f-3f615c4030cc.png)